src/converters/array-converter.js

```diff
@@ -8,6 +28,17 @@ var list = [
         }
     },
     {
+        /**
+         * Select and display only the last element of the array.
+         *
```

Comment:
Would just say "select" instead "select and display" (displaying things is the responsibility of the dom not the converter)